Lib/ufoLib/glifLib.py

```diff
@@ -128,17 +128,17 @@ def rebuildContents(self):
 		Rebuild the contents dict by loading contents.plist.
 		"""
 		contentsPath = os.path.join(self.dirName, "contents.plist")
-		if not os.path.exists(contentsPath):
+		try:
+			contents = self._readPlist(contentsPath)
+		except KeyError:
```

Comment:
Could we use a different exception class here and elsewhere you used KeyError? That’s a bit too generic and may leak from any other dict lookup. Sublassing GlifLibError is cheap